### ISSUE TITLE
Error when loading Sinopia Resource

### ISSUE THREAD

**Issue by jgreben** [LD4P/sinopia_editor#1511]
On Sinopia Stage:

search Sinopia for “Virgile” and select “Virgile travestie, Scarronnides…” 
Nothing happens but the console logs this error:
```
More than one resource template matches: ld4p:RT:bf2:Form,ld4p:RT:bf2:RareMat:RBMS,ld4p:RT:bf2:RareMat:AAT
```

![image](https://user-images.githubusercontent.com/3093850/66520798-b42b9c00-ea9e-11e9-9059-f123e248ba6a.png)

On the Sinopia Slack channel some people mentioned different buggy behavior with this resource, including:

- `when I opened it, I saw a “Map of Longmont and adjoining additions”...There’s also a weird delay when opening a record where… after the last record I opened is displayed, then the screen blinks and the right one shows`
- `it just opens in whatever the last template I was using and there is no data at all.`
- `I'm getting a completely different resource`

        ¯\_(ツ)_/¯ 

It looks like if a RT was loaded prior to doing the search it will show the previous template with no data, but if nothing was loaded prior to selecting the search, it will do nothing. In all cases the console error is show.

**Comment by jgreben** [comment]
Error is thrown in https://github.com/LD4P/sinopia_editor/blob/0295700b98e290464f2518395724309e6ba74ae6/src/ResourceStateBuilder.js#L151

**Comment by michelleif** [comment]
may be related to #1447

**Comment by justinlittman** [comment]
I suspect this is addressed by #1447. Can the problem be replicated in dev? Can this wait until next release to stage in 1 week or should we release earlier?

**Comment by michelleif** [comment]
cannot reproduce with the specific Resource since it doesn't exist in DEV, but I just re-tested #1447 on DEV and it is now showing users error messages when templates are missing. I would rather release this sooner as it is pretty disruptive and leading users to think other things are messed up.

**Comment by michelleif** [comment]
fixed but waiting on #1522
